packages/xo-web/src/xo-app/backup/health/index.js

```diff
@@ -4,14 +4,84 @@ import Icon from 'icon'
 import Link from 'link'
 import NoObjects from 'no-objects'
 import React from 'react'
-import renderXoItem from 'render-xo-item'
+import renderXoItem, { Vm } from 'render-xo-item'
 import SortedTable from 'sorted-table'
-import { addSubscriptions, connectStore } from 'utils'
+import { addSubscriptions, connectStore, noop } from 'utils'
 import { Card, CardHeader, CardBlock } from 'card'
+import {
+  compact,
+  filter,
+  flatMap,
+  forEach,
+  isEqual,
+  keyBy,
+  map,
+  omit,
+  toArray,
+} from 'lodash'
 import { Container, Row, Col } from 'grid'
+import { confirm } from 'modal'
+import { createPredicate } from 'value-matcher'
 import { createGetLoneSnapshots, createGetObjectsOfType } from 'selectors'
-import { deleteSnapshot, deleteSnapshots, subscribeSchedules } from 'xo'
-import { FormattedRelative, FormattedTime } from 'react-intl'
+import {
+  deleteBackups,
+  deleteSnapshot,
+  deleteSnapshots,
+  listVmBackups,
+  subscribeBackupNgJobs,
+  subscribeRemotes,
+  subscribeSchedules,
+} from 'xo'
+import { FormattedDate, FormattedRelative, FormattedTime } from 'react-intl'
+
+const DETACHED_BACKUP_COLUMNS = [
+  {
+    name: _('date'),
+    itemRenderer: backup => (
+      <FormattedDate
+        value={new Date(backup.timestamp)}
+        month='long'
+        day='numeric'
+        year='numeric'
+        hour='2-digit'
+        minute='2-digit'
+        second='2-digit'
+      />
+    ),
+    sortCriteria: 'timestamp',
+    sortOrder: 'desc',
+  },
+  {
+    name: _('vm'),
+    itemRenderer: ({ vmId }) => <Vm id={vmId} link />,
+    sortCriteria: ({ vmId }, { vms }) => {
+      const vm = vms[vmId]
+      return vm !== undefined && vm.name_label
+    },
+  },
+  {
+    name: _('job'),
+    itemRenderer: ({ jobId }, { jobs }) => {
+      const job = jobs[jobId]
+      return job === undefined ? null : (
+        <Link to={`/backup/overview?s=id:${jobId}`}>{job.name}</Link>
+      )
+    },
+    sortCriteria: ({ jobId }, { jobs }) => {
+      const job = jobs[jobId]
+      return job !== undefined && job.name
+    },
+  },
+  {
+    name: _('jobModes'),
+    itemRenderer: ({ mode }) => mode,
+    sortCriteria: 'mode',
+  },
+  {
+    name: _('reason'),
+    itemRenderer: backup => backup.reason,
```

Comment:
The internationalization can be done here to be able to sort this column.

Reply:
We can also translate on the display, to be able to manipulate the reason. IMHO, translating on the display has more advantages than translating before the display.